app/assets/javascripts/events_form.js

```diff
@@ -0,0 +1,95 @@
+var i = 0
+
+// Function makes the edit window show single form and hides file input
+window.onload= () =>{
+  if(window.location.href.indexOf("edit") > -1 ) {
+    document.getElementById('switchButton').classList.toggle("hidden")
+    document.getElementById("instructions").innerHTML = ""
+
+    // Hide unnecessary information
+    event_repeatmonday_input= document.getElementById("event_repeatmonday_input")
+    event_repeatmonday= document.getElementById("event_repeatmonday")
+    event_repeatmonday_input.classList.toggle("hidden")
+	
+	event_repeattuesday_input= document.getElementById("event_repeattuesday_input")
```

Comment:
kind of nit picking but can you fix the tabbing throughout this file

Reply:
Done.